Jantimizer/QueryTestSuite/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using QueryTestSuite.Connectors;
+using QueryTestSuite.Models;
+using System.Data;
+
+namespace QueryTestSuite
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Task.WaitAll(AsyncMain(args));
+        }
+
+        async static Task AsyncMain(string[] args)
+        {
+            var pgConn = new PostGres("Host=localhost;Username=postgres;Password=Aber1234;Database=JanBase");
+            var myConn = new MySql("Host=localhost;Username=Jantimizer;Password=Aber1234");
```

Comment:
Why the username? Should it not use default?
```suggestion
            var pgConn = new PostGres("Host=localhost;Username=postgres;Password=password;Database=JanBase");
            var myConn = new MySql("Host=localhost;Username=root;Password=password");
```

Reply:
I made an issue regarding this #11.
Something that can be changed later.